public/app/features/dashboard-scene/panel-edit/PanelOptions.test.tsx

```diff
@@ -116,7 +116,7 @@ describe('PanelOptions', () => {
 
```

Comment:
Changes in this and the next file seem unrelated?

Reply:
Hey @sunker! Option pane Title field selector was changed so in `e2e/plugin-e2e/plugin-e2e-api-tests/as-admin-user/panelEditPage.spec.ts` `edit panel plugin settings` broke. Adjusted selector to the one test expects 👍